src/discovery/discovery.service.ts

```diff
@@ -44,11 +45,9 @@ export class DiscoveryService {
 
   async coloredFilteredValidation(issuer: object, schema_file: string, keys: any[]) {
       const [ valid, errors ] = await this.validateJson(issuer, schema_file);
-      console.log('start validation');
       if (valid) {
-          const schema = require('../../schema/' + schema_file);
+          const schema = require(join('..', '..', 'schema', schema_file));
```

Comment:
This results in an internal error, when no schema_file is specified. Maybe check if it is even necessary to load this information.
`[Nest] 18952  - 14.06.2022, 18:27:46   ERROR [ExceptionsHandler] Cannot find module '../../schema/token'`

Reply:
I added a check if schema_s is null